leshan-core/src/main/java/org/eclipse/leshan/core/model/ObjectLoader.java

```diff
@@ -107,6 +105,52 @@ public static ObjectModel loadDdfFile(InputStream input, String streamName) {
         return models;
     }
 
+    /**
+     * Load object definition from DDF files.
+     * 
+     * @param path directory path to the DDF files
+     * @param filenames names of all the DDF files
+     */
+    public static List<ObjectModel> loadDdfFiles(String path, String[] filenames) {
+        List<ObjectModel> models = new ArrayList<>();
+        for (String filename : filenames) {
+            String fullpath = path + filename;
```

Comment:
We use `class.getResourceAsStream` so `/` should be used not `\` (see [javadoc](https://docs.oracle.com/javase/7/docs/api/java/lang/ClassLoader.html#getResource%28java.lang.String%29)).

Maybe our javadoc is not clear, we should explain that we use `class.getResource()` and point the javadoc which explain this is for loading resource _in a way that is independent of the location of the code._ (loading resource in jar or war file)

About resilience of trailing `/`, I can add it but I found this is not so usefull. This method is more for loading statically resource you embedded in your jar or war. So you will use constant and quickly see if you can not load your files. (because of illegalStateException)

If you want something more dynamic (the case you don't embedded your models in your jar) you can use [loadObjectsFromDir](https://github.com/eclipse/leshan/pull/299/files/54053b03413fcbc4361ffda5b9dbe40216198581#diff-449a4a63d5ac98e293fecf026aaf8165R157) which should be public now.